metals/src/main/scala/scala/meta/internal/worksheets/WorksheetProvider.scala

```diff
@@ -291,18 +292,31 @@ class WorksheetProvider(
   }
 
   private def getMdoc(target: BuildTargetIdentifier): Option[Mdoc] = {
+
+    def isSupportedScala3Version(scalaVersion: String) = {
+      ScalaVersions.isScala3Version(scalaVersion) && SemVer.isCompatibleVersion(
+        "0.26.0",
```

Comment:
Can we include a note either here or in the commit to why it has to be 0.26.0 or newer.